xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -174,6 +200,38 @@ void CVideoLayerBridgeDRMPRIME::Configure(CVideoBufferDRMPRIME* buffer)
     m_DRM->AddProperty(plane, "COLOR_RANGE", value);
 
   auto connector = m_DRM->GetConnector();
+
+  int bpc = 8;
+  bool ycc = false;
+  std::string output_format = "Default";
+
+  if (picture.colorBits > 8)
+  {
+    bpc = 12;
```

Comment:
How do we know to use this value? What about 10 or 14, or 16 or 10000?

Reply:
The new patches will just choose the highest bpc it can achieve, and will go down from that if we can't. So it will start at 12, try RGB and YUV422, if both fail, 10, and if it fails again 8.

The userspace can limit it if needed using the max_bpc property, but if it doesn't care it will happen automatically.